go.mod

```diff
@@ -3,7 +3,10 @@ module zuri.chat/zccore
 go 1.16
 
 require (
+	github.com/0xAX/notificator v0.0.0-20210731104411-c42e3d4a43ee // indirect
```

Comment:
Where is this coming from? I didn't see it usage in the tests you wrote

Reply:
I don't know